utils/datasets/BUILD

```diff
@@ -73,7 +81,8 @@ py_test(
     srcs = ["stackoverflow_word_prediction_test.py"],
     python_version = "PY3",
     srcs_version = "PY3",
-    deps = [":stackoverflow_word_prediction"],
+    deps = [":stackoverflow_word_predi
+ction"],
```

Comment:
Why do you need to change this file. I think it is complaining about this new line. Could you discard your change in utils/dataset/? Thanks!

Reply:
Sorry about that! Made the correction.